functions/test/wort/core_test.clj

```diff
@@ -4,4 +4,13 @@
 
 (deftest a-test
   (testing "FIXME, I fail."
-    (is (= 0 1))))
+    (is (= 1 1))))
+
+(deftest _get_words_from_key
```

Comment:
It namespace clashes with the core method get_words, I'm trying to figure out how to alias it /  how namespace creation works in clojure. On it.